### DISCUSSION TITLE
Choosing how my C# is formatted

### DISCUSSION THREAD

**Discussion by ghost** [muhammadsammy/free-omnisharp-vscode#66] [Q&A] [answered]
Hi, I'm using VSCodium to develop in C# and I've selected this as the code formatter, but I can't see a way of altering the formating. For instance I know I'm weird, but I want my code formatted like this:
`public void Something() {
    //without a line break for the open brace

}`

any help?

**Comment by muhammadsammy** [comment, marked as answer]
Try one of the answers here: https://stackoverflow.com/questions/49136015/vscode-format-curly-brackets-on-the-same-line-c-sharp
